### PR TITLE
Added design layout without adaptive design

### PR DESCRIPTION
**Pull request by AlexKrasnova**

### CODE REVIEW

**Review by miwanoff — COMMENTED** [review]
1. Styling of separate files SVG cannot be changed at style.css, so there are two way of recolor them:  
a) at svg file like this: &lt;path d="..." fill="#ffc373"/&gt; 
b) copy the code of this file to the html html. 
I prefer the first way.
2. With developer tools we can see styles of the tr element like this:
.table-striped>tbody>tr:nth-of-type(odd)>* {
    --bs-table-accent-bg: var(--bs-table-striped-bg);
    color: var(--bs-table-striped-color);
}
so we would change color this way: 
.table-striped>tbody>tr:nth-of-type(odd)>* {
    --bs-table-accent-bg: white;
}
3. And we can find below style with border-style:  
tbody, td, tfoot, th, thead, tr {
    border-color: inherit;
    border-style: solid;
    border-width: 0;
} 
- td is bold, so we need to override css style for td: 
td {
  border-style: none;
}

File: css/style.css [hunk 1]
```diff
@@ -1,3 +1,188 @@
...
+}
+
+.table-striped>tbody>tr:nth-child(even)>td, 
+.table-striped>tbody>tr:nth-child(even)>th {
+    background-color: var(--corgi-color-fill-15);
+}
+
+.table-striped>tbody>tr:nth-child(odd)>td, 
+.table-striped>tbody>tr:nth-child(odd)>th{
+   background-color: white !important;
```
Comment (miwanoff) [inline css/style.css:157]: This doesn't work

File: css/style.css [hunk 2]
```diff
@@ -1,3 +1,188 @@
...
+ }
+
```
Comment (miwanoff) [inline css/style.css:159]: .table-striped>tbody>tr:nth-of-type(odd)>* {
--bs-table-accent-bg: white;
}

File: css/style.css [hunk 3]
```diff
@@ -1,3 +1,188 @@
...
+ .table a {
+    text-decoration: none;
+    color: black;
+ }
+
```
Comment (miwanoff) [inline css/style.css:164]: td {
border-style: none;
}

Comment (miwanoff) [reply css/style.css:164]: th, td {
border-style: none;
}

File: css/style.css (hunk 1 above)
Comment (AlexKrasnova) [reply css/style.css:157]: I know. But why did it work for even rows? And doesn't work for odd? Or did I just added background color for even rows and didn't replace the original white background?

**Review by professor-pirog — APPROVED** [review]

**Pull request merged**